libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/BaseTest.kt

```diff
@@ -54,4 +56,17 @@ open class BaseTest<A : AppCompatActivity>(activityClass: Class<A>) {
 
     val activity: A
         get() = activityRule.activity
+
+    @Before
+    fun runSetupMockLocation() {
+        mockLocationUpdatesRule.pushLocationUpdate(setupMockLocation())
```

Comment:
Should this `runOnMainSync`? 🤔

Reply:
I don't think this should be needed, mostly because all "before" functions are completed before the functional test code has started.

There could be an issue if a downstream test tries to initialize the current location in the before block. Also checked the android source and it is [synchronizing location updates](https://android.googlesource.com/platform/frameworks/base/+/android-4.3_r2.1/services/java/com/android/server/LocationManagerService.java#2075 )